include/cartesian_geom/point.h

```diff
@@ -0,0 +1,105 @@
+#ifndef POINT_H
+#define POINT_H
+
+
+template <class Cr>
+class point
+{
+public:
+    typedef typename Cr::RT 	K;
+    int d;
+    typedef std::vector<K> coeff;
+    coeff coeffs;
+    
+    typedef typename std::vector<K>::iterator iter;
+    
+    point() {}
+    
+    point(int dim){
```

Comment:
dim is `const`, similarly use `const` whenever possible in the whole class

Reply:
done